incubator-tools/date_entities_annotation_tool/date_entities_annotation.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_token(json_dict : object, page : str, text_anchors_check : list ) -&gt; tuple[dict,list,float] :
</pre><p>Lots of repeated code that could be refactored into a function.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>